test/data/dictionary-epub-chapters/ch06_direct_lookup.html

```
@@ -0,0 +1,138 @@
+<li>Select <em>chronicle</em>. Definition screen appears on page 1.</li>
+<li>Press <em>PageForward</em> or <em>Right</em>. Page 2 appears.</li>
+<li>Press <em>PageBack</em> or <em>Left</em>. Page 1 reappears.</li>
+<li>Long-press <em>Back</em>. Reader returns.</li>
+</ol>
+
+<h2>Test C: Definition viewer honors reader font size</h2>
+<p>The definition viewer must use the same font size as the reader. Changing the
+font size in Settings must be reflected in the definition screen.</p>
+<h3>Steps</h3>
+<ol>
+<li>Note the current font size in Settings -> Reader -> Font Size.</li>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)